workshop-service/workshop.go

```
@@ -38,6 +49,12 @@ func postWorkshopHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if !isValidSweaterScore(newWorkshop.SweaterScore) {
+		w.WriteHeader(http.StatusBadRequest)
+		w.Write([]byte("SweaterScore must be between 1 and 10"))
```

golangci-lint (1.62.2)

Error return value of `w.Write` is not checked

(errcheck)